FakebookNotifications/FakebookNotifications.DataAccess/NotificationsContext.cs

```diff
@@ -78,16 +84,17 @@ public bool ClearNotifications(IMongoCollection<User> userCol, IMongoCollection<
 
                 return true;
             }
-            catch(Exception ex)
+            catch
             {
-                Debug.WriteLine("Could not clear previous seed data" + ex);
+                _logger.LogWarning("Unable to clear seed data");
```

Comment:
half the point of logging here is to log the exception, use the overload that includes the exception with the string message